### PR TITLE
[risk=low][no ticket] rm all traces of v1 Demo Survey from UI

### PR DESCRIPTION
**Pull request by jmthibault79**
The "enable v2 demo survey" feature flag has been true in all environments for a long time.  Let's remove it, as well as Demo Survey V1.

This is the UI step to doing so.  One release later, we can then remove it from the API as well.

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have added explanatory comments where the logic is not obvious
- [ ] I have run and tested this change locally, and my testing process is described here
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later
- [ ] If this includes an API change, I have run the E2E tests on this change against my local server with [yarn test-local](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples) because this PR won't be covered by the CircleCI tests 
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P)
- [ ] If this change impacts deployment safety (e.g. removing/altering APIs which are in use) I have documented these in the description
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and updated the appropriate API consumers
  * AoU UI
  * [Perf tests](https://github.com/broadinstitute/mcnulty/blob/develop/src/test/scala/services/AoU.scala)
  * [Researcher Directory export](https://github.com/all-of-us/workbench/wiki/Researcher-Directory-(RDR-export))
  * Cron tasks - for Offline*Controllers
  * SumoLogic - for EgressAlert

### CODE REVIEW

File: ui/src/app/pages/profile/demographic-survey-panel.tsx [hunk 1]
```diff
@@ -1,62 +1,43 @@
...
 import { DEMOGRAPHIC_SURVEY_V2_PATH } from 'app/utils/constants';
 import { displayDateWithoutHours } from 'app/utils/dates';
-import { serverConfigStore } from 'app/utils/stores';
 
 import { styles } from './profile-styles';
 
 interface Props {
   demographicSurveyCompletionTime: number;
-  firstSignInTime: number;
   onClick: () => void;
```
Comment (evrii) [inline ui/src/app/pages/profile/demographic-survey-panel.tsx:11]: Is this needed?

**Review by evrii — APPROVED** [review]
Looks good to me. Ran tests locally to confirm the changes worked as intended.

File: ui/src/app/pages/profile/demographic-survey-panel.tsx (hunk 1 above)
Comment (jmthibault79) [reply ui/src/app/pages/profile/demographic-survey-panel.tsx]: good catch.  I can actually remove a lot more - like the whole v1 Demo Survey

**Comment by jmthibault79** [comment]
This PR completes https://precisionmedicineinitiative.atlassian.net/browse/RW-8365

**Pull request merged**